docs/design/autoware-interfaces/components/planning.md

```diff
@@ -1,152 +1,71 @@
 # Planning
 
-![Node diagram](images/Planning-Bus-ODD-Architecture.drawio.svg)
+This page provides specific specifications about the Interface of the Planning Component. Please refer to the [planning architecture design document](../autoware-architecture/planning/index.md) for high-level concepts and data flow.
```

Comment:
This link does not work.

Reply:
Oops.

Reply:
solved in https://github.com/autowarefoundation/autoware-documentation/pull/387/commits/7dc14f02c350d0b6a3c0414d063a391a016340bd !